yoti-sdk-spring-boot-example/src/main/java/com/yoti/api/examples/springboot/YotiLoginController.java

```diff
@@ -114,7 +117,7 @@ private DisplayAttribute mapToDisplayAttribute(Attribute attribute) {
             case AttributeConstants.HumanProfileAttributes.POSTAL_ADDRESS:
                 return new DisplayAttribute("Address", attribute, "yoti-icon-address");
             case AttributeConstants.HumanProfileAttributes.STRUCTURED_POSTAL_ADDRESS:
-                return null; // Do nothing - we are handling this with the postalAddress attribute
+                return new DisplayAttribute("Structured Postal Address", attribute, "yoti-icon-address"); // Do nothing - we are handling this with the postalAddress attribute
```

Comment:
Can remove the comment now:
```suggestion
                return new DisplayAttribute("Structured Postal Address", attribute, "yoti-icon-address");
```